### PR TITLE
add tufte-memo:0.1.0

### PR DESCRIPTION
**Pull request by nogula**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: this template package provides a memo document template inspired by the design of Edward Tufte's books.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by nogula** [comment]
First time submitter (and not an avid Git user, either)... the checks fail on account of not finding the readme.md file, but this file is certainly present, from commit [22718be](https://github.com/typst/packages/pull/713/commits/22718be256dbad487ad008f059a7930a873d0ede) and also seen in [my fork](https://github.com/nogula/packages/blob/main/packages/preview/tufte-memo/0.1.0/readme.md). Would someone be so kind as to point me in the right direction?

**Comment by nogula** [comment]
Ah. Case sensitive.

File: packages/preview/tufte-memo/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,138 @@
+// #import "@preview/tufte-memo:0.1.0": *
+#import "@local/tufte-memo:0.1.0": *
```
Comment (elegaanz) [inline packages/preview/tufte-memo/0.1.0/template/main.typ:2]: This should use the `@preview` import:

```suggestion
#import "@preview/tufte-memo:0.1.0": *
```

File: packages/preview/tufte-memo/0.1.0/lib.typ [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    (
+      name: "First Last",
+      role: none,
+      organization: none,
+      location: none,
+      email: none
+    ),
+  ),
+  date: datetime.today(),
+  document_number: none,
```
Comment (elegaanz) [inline packages/preview/tufte-memo/0.1.0/lib.typ:33]: To be consistent with the rest of the package and of the Typst ecosystem, this variable should probably use kebab-case and be renamed to `document-number`.

**Comment by nogula** [comment]
@elegaanz I've addressed your comments in 30a3616 - thanks.

**Pull request merged**